mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/impl/HibernateOrmMassIndexingContext.java

```diff
@@ -173,10 +173,12 @@ public PojoMassIdentifierLoader createIdentifierLoader(PojoMassIndexingIdentifie
 					.collect( Collectors.toList() );
 
 			HibernateOrmQueryLoader<E, I> typeQueryLoader = createQueryLoader( typeContexts );
+			// NOTE: we are creating a regular and not a stateless session,
```

Comment:
Hmm. I have no idea what kind of impact this could have... Given we only execute a scroll on identifiers, I'd hope not much, but who knows...
I think we should be very careful about backporting this. Ideally we would not, and would backport the fix on the ORM side instead.

Reply:
yeah, I am 50-50 on the change... if we get a fix into ORM, then we won't need the change in Search...